readthedocs/doc_builder/python_environments.py

```diff
@@ -215,10 +213,8 @@ def install_core_requirements(self):
 
         # Install pip-only things.
         pip_requirements = [
-            'mkdocs==0.15.0',
-            'readthedocs-sphinx-ext<0.6',
-            'commonmark==0.5.4',
-            'recommonmark==0.1.1',
+            'readthedocs-sphinx-ext',
+            'recommonmark',
```

Comment:
I do worry about unpinning these. @agjohnson would know best, but I believe there is a released version that might bork things.